src/webextension/manage/containers/add-item.js

```diff
@@ -9,11 +9,17 @@ import { connect } from "react-redux";
 
 import Button from "../../widgets/button";
 import { startNewItem } from "../actions";
+import * as telemetry from "../../telemetry";
 
 function AddItem({dispatch}) {
+  const doClick = () => {
+    telemetry.recordEvent("lockbox", "addClick", "addButton");
+    dispatch(startNewItem());
+  };
+
   return (
-    <Localized id="toolbar-add-item">
```

Comment:
This missing "toolbar-" prefix probably needs to get added in based on the latest UX.

Reply:
Good catch; it caught up in a conflict.